Makefile

```diff
@@ -6,10 +6,15 @@ SKOPEO_BRANCH = master
 # Set SUDO=sudo to run container integration tests using sudo.
 SUDO =
 
+ifneq ($(GOOS),linux)
```

Comment:
Maybe optimize for clarity instead of number of lines?
```
DARWIN_NATIVE_TAGS = …ostree…
DARWIN_CROSS_TAGS = …ostree… …openpgp…
WINDOWS_CROSS_TAGS =…

…

ifeq ($(shell uname),Darwin)
   PLATFORM_BUILD_TAGS = $(DARWIN_NATIVE_TAGS)
endif

…

build: vendor build-internal

build-internal:
	@$(GPGME_ENV) go build $(BUILDFLAGS) $(PACKAGES)

cross:
	make build-internal GOOS=windows PLATFORM_BUILD_TAGS="$(WINDOWS_CROSS_TAGS)"
	make build-internal GOOS=darwin PLATFORM_BUILD_TAGS="$(DARWIN_CROSS_TAGS)"
```

(or with `go build` instead of the new `build-internal` target; all completely untested)

Reply:
i'll give that a try